src/components/MoviesList/MoviesList.jsx

```diff
@@ -1,10 +1,23 @@
 import './MoviesList.scss';
 import { MovieCard } from '../MovieCard';
 
-export const MoviesList = ({ movies }) => (
-  <div className="movies">
-    {movies.map(movie => (
-      <MovieCard key={movie.imdbId} movie={movie} />
-    ))}
-  </div>
-);
+export const MoviesList = ({ movies, query }) => {
+  let preparedMovies = [...movies];
```

Comment:
```suggestion
  const preparedMovies = getPreparedMovies();
```
Try to avoid reassigning the variable. Filter the movies in place or create a function for that

Reply:
Not fixed